frontend/src/pages/SettingsRouter/SettingsRouter.tsx

```diff
@@ -304,6 +314,10 @@ export const SettingsRouter = () => {
 										/>
 									)
 								})}
+								<Route
+									path="integrations"
+									element={<IntergationPage />}
+								/>
```

Comment:
let's remove the https://app.highlight.io/1/integrations page since we now have https://app.highlight.io/w/1/integrations/clickup. can we have /1/integrations redirect to /w/1/integrations/ to preserve existing docs?

please remove the link to the old integrations and just keep the new one in the workspace settings
![Screenshot from 2024-12-09 16-34-18](https://github.com/user-attachments/assets/751501f3-aabd-4415-96bf-e09ef419d0a5)